src/main/java/com/renatusnetwork/momentum/commands/CommandSignCMD.java

```diff
@@ -38,70 +39,149 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
+                break;
+            case "title":
+                if (!csignManager.commandSignExists(name)) {
+                    sender.sendMessage(Utils.translate("&cNo command sign exists with name &4" + name));
+                    return true;
+                }
+
+                String title = String.join(" ", Arrays.copyOfRange(a, 2, a.length));
+                csignManager.updateTitle(name, title);
+                sender.sendMessage(Utils.translate("&7Set title for &2" + name + "&7 to &a" + title));
+                break;
+            case "show":
+                if (!csignManager.commandSignExists(name)) {
+                    sender.sendMessage(Utils.translate("&cNo command sign exists with name &4" + name));
+                    return true;
+                }
+
+                CommandSign csign = csignManager.getCommandSign(name);
+                showInfo(sender, csign);
+                break;
+            case "add":
+                if (!csignManager.commandSignExists(name)) {
+                    sender.sendMessage(Utils.translate("&cNo command sign exists with name &4" + name));
+                    return true;
+                }
+
+                cmd = String.join(" ", Arrays.copyOfRange(a, 2, a.length));
+                if (cmd.isEmpty()) {
+                    sendHelp(sender);
+                    return true;
+                }
+
+                csignManager.addCommand(name, cmd);
+                sender.sendMessage(Utils.translate("&aSuccessfully added command to &2" + name));
+                break;
+            case "remove":
+                if (a.length < 3) {
+                    sendHelp(sender);
+                    return true;
+                }
+
+                if (!csignManager.commandSignExists(name)) {
+                    sender.sendMessage(Utils.translate("&cNo command sign exists with name &4" + name));
+                    return true;
+                }
+
+                if (a[2].equalsIgnoreCase("all")) {
+                    csignManager.clearCommands(name);
+                    sender.sendMessage(Utils.translate("&aSuccessfully removed all commands for &2" + name));
+                    break;
+                }
+
+                try {
+                    index = Integer.parseInt(a[2]) - 1; // minus one since the displayed list in game is shifted to start at 1 instead of 0 for practicality
+                } catch (NumberFormatException ignore) {
+                    sendHelp(sender);
+                    return true;
+                }
+
+                result = csignManager.removeCommand(name, index);
+                sender.sendMessage(result ? Utils.translate("&aSuccessfully removed command at index &2" + (index + 1) + " &afor &2" + name) : Utils.translate("&cindex &4" + (index + 1) + " &cdoes not exist for &4" + name));
```

Comment:
We can reverse this to

`sender.sendMessage(Utils.translate(result ? "&aSuccessfully removed command at index &2" + (index + 1) + " &afor &2" + name : "&cindex &4" + (index + 1) + " &cdoes not exist for &4" + name))`